examples/appendix_a/main.rs

```diff
@@ -20,9 +20,9 @@ use amethyst::{
     ui::UiBundle,
     utils::application_root_dir,
 };
-use audio::Music;
-use bundle::PongBundle;
-use config::PongConfig;
+use crate::audio::Music;
+use crate::bundle::PongBundle;
+use crate::config::PongConfig;
```

Comment:
Shouldn't all of these be grouped imports? :thinking:

Reply:
yeah I suppose haha, both of the automated tools failed to catch that but we can!

Reply:
There are more instances of this than just here btw!

Reply:
I think I got them all.